packages/android_studio.rb

```diff
@@ -15,17 +15,17 @@ class Android_studio < Package
 
   def self.preflight
     free_space = `echo $(($(stat -f --format="%a*%S" #{CREW_PREFIX})))`.chomp.to_i
-    abort 'Not enough free disk space.  You need at least 6 GB to install.'.lightred if free_space < 6442450944
+    abort 'Not enough free disk space.  You need at least 6 GB to install.'.lightred if free_space < 6_442_450_944
```

Comment:
This is wrong.  Should be a number with no placeholders.

Reply:
I've already disabled Naming/VariableNumber, so not sure what is causing this. Maybe a rubocop bug?

Reply:
@Zopolis4 Can you get this taken care of?